test/v1beta1/note_test.go

```diff
@@ -1,7 +1,54 @@
 package v1beta1_test
 
-import "testing"
+import (
+	fake "github.com/brianvoe/gofakeit/v5"
+	"github.com/grafeas/grafeas/proto/v1beta1/build_go_proto"
+	"github.com/grafeas/grafeas/proto/v1beta1/common_go_proto"
+	"github.com/grafeas/grafeas/proto/v1beta1/grafeas_go_proto"
+	"github.com/liatrio/grafeas-elasticsearch/test/util"
+	. "github.com/onsi/gomega"
+	"testing"
+)
 
 func TestNote(t *testing.T) {
-	t.Skipf("TODO")
+	Expect := util.NewExpect(t)
+	s := util.NewSetup()
+
+	// setup project for occurrences
+	projectName := util.RandomProjectName()
+
+	// generate note ID (note IDs are provided by client, not generated by server)
+	noteId := fake.UUID()
```

Comment:
This seems like something that should be different for every test case right? Not sure if should be defined once top-level.

Reply:
oops, I was writing this as if it was in a `BeforeEach`